src/Contracts/Form/index.js

```diff
@@ -122,6 +128,34 @@ const Form = props => {
   )
 }
 
+// NOTE copied from web3 v1.0 isAddress. Remove when we upgrade to web3 v1.0
+const isAddress = address => {
+  if (!/^(0x)?[0-9a-f]{40}$/i.test(address)) {
+    // check if it has the basic requirements of an address
+    return false
+  } else if (/^(0x)?[0-9a-f]{40}$/.test(address) || /^(0x)?[0-9A-F]{40}$/.test(address)) {
+    // If it's all small caps or all all caps, return true
+    return true
+  } else {
+    // Otherwise check each case
+    return isChecksumAddress(address)
+  }
+}
+
+// NOTE copied from web3 v1.0 isAddress. Remove when we upgrade to web3 v1.0
+const isChecksumAddress = address => {
+  // Check each case
+  address = address.replace('0x', '')
+  const addressHash = sha3(address.toLowerCase())
+  for (let i = 0; i < 40; i++) {
+    // the nth letter should be uppercase if the nth digit of casemap is 1
+    if ((parseInt(addressHash[i], 16) > 7 && address[i].toUpperCase() !== address[i]) || (parseInt(addressHash[i], 16) <= 7 && address[i].toLowerCase() !== address[i])) {
+      return false
+    }
+  }
+  return true
+}
```

Comment:
you can use `ethers.js` for that

Reply:
do you know which function offhand? I don't see it in the docs

Reply:
yes you're right. Later we can put these functions in a `utils` folder